contrib/charts/cert-manager/templates/hook_apply_issuers.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.clusterIssuer.email }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: apply-issuers
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      name: apply-issuers
+      namespace: {{ .Release.Namespace }}
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: apply-issuers
+        image: claranet/gcloud-kubectl-docker
+        imagePullPolicy: Always
+        command: ["kubectl","apply","-f","/tmp/"]
```

Comment:
Running `kubectl apply` on upgrade is a bit dangerous; apply is prone to delete user's changes if the kubectl version mismatches enough, and with CRDs it has had numerous issues in the past.

I worry that this naive upgrade could break people if they've diverged too far from upstream... and there's no annotation (e.g. `managed-by: cert-manager-helm`) that the user can delete/edit to opt-out of upgrades attempting to migrate it.

One rather simple example: I have `http01: {}` turned off for my cluster issuer. This would try to turn it back on for the default one each time.